knowledge-graph/src/main/scala/io/renku/knowledgegraph/MicroserviceRoutes.scala

```diff
@@ -87,9 +90,23 @@ private class MicroserviceRoutes[F[_]: Async](
     (versionRoutes() <+> nonAuthorizedRoutes <+> authorizedRoutes).withMetrics
 
   private lazy val authorizedRoutes: HttpRoutes[F] = authMiddleware {
-    `GET /datasets/*` <+> `GET /entities/*` <+> `GET /projects/*` <+> `GET /users/*`
+    `GET /datasets/*` <+> `GET /entities/*` <+> `GET /projects/*` <+> `GET /users/*` <+> getRecentEntities
   }
 
+  private lazy val getRecentEntities: AuthedRoutes[Option[AuthUser], F] =
```

Comment:
I guess we'd need to use the new `MaybeAuthUser` here but the compiler will complain anyway.

Reply:
From what I understood here is that the user is required, right. So it probably needs to be `AuthUser` then, thanks!